test/unit/controllers/declaration/AdditionalInformationControllerSpec.scala

```diff
@@ -197,8 +198,8 @@ class AdditionalInformationControllerSpec extends ControllerSpec with ErrorHandl
 
         val result = controller.saveAdditionalInfo("itemId")(postRequestAsFormUrlEncoded(correctForm: _*))
 
-        status(result) must be(SEE_OTHER)
-        redirectLocation(result) must be(Some(routes.DocumentsProducedController.displayPage("itemId").url))
+        await(result) mustBe aRedirectToTheNextPage
```

Comment:
Why await? Why we're checking this differently? 

If we want to do it I prefer to extends ScalaFutures and use `.futureValue` on result. This comment is for all changes like this below

Reply:
Why not use await? :)

We are checking it differently because the result is returned from the navigator which is mocked

Reply:
In hindsight there is nicer ways to assert on this e.g. by mocking the navigator so it returns a `Redirect(the route that was passed in)`

But Paulo is working on this too so I don't want to cause all conflicts for him

Reply:
ah, okay.

Probably `await` is from hmrctest, as I remember, that is deprecated (repo not method), so it's good to avoid more places with this, because we need to ged rid of this library. Moreover, `futureValue` is from ScalaFutures. It's part of scalatest, so IMO it's better choice, but the second answer is related with my personal thinking, so it's not something that must be or not

Reply:
It's from `play.api.test.Helpers` I believe not hmrctest

Although I think theres a similar one in hmrctest

Reply:
@PatrykRudnicki await is also from play-test-helpers